hphp/hack/src/errors/typing_error.ml

```diff
@@ -1179,7 +1179,19 @@ module Primary = struct
           async_indicator
           return_type
       in
-      (Error_code.NonVoidAnnotationOnReturnVoidFun, (pos, message), [], [])
+      let quickfix =
```

Comment:
I might be tempted to call this `quickfixes` as it's a list of quickfixes, not a single value.